server/datastore/mysql/locks.go

```diff
@@ -54,7 +54,8 @@ func (ds *Datastore) overwriteLockIfExpired(ctx context.Context, name string, ow
 }
 
 func (ds *Datastore) Unlock(ctx context.Context, name string, owner string) error {
-	_, err := ds.writer.ExecContext(ctx, `DELETE FROM locks WHERE name = ? and owner = ?`, name, owner)
+	_, err := ds.writer.ExecContext(ctx, `UPDATE locks SET expires_at = CURRENT_TIMESTAMP WHERE name = ? AND owner = ?`, name, owner)
```

Comment:
So I guess we'll have to cleanup the locks table at some point since we are not removing the items anymore.

Reply:
As it stands, the locks table should only ever have one row per schedule so I don't think clean up would be a big issue.